<?xml version="1.0"?>
<launch>
  <include file="$(find wr_pi_extend_board_driver)/launch/core.launch" />
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
        <param name="video_device" value="/dev/video0"/>
        <param name="image_width" value="320"/>
        <param name="image_height" value="240"/>
        <param name="pixel_format" value="yuyv"/>
        <param name="framerate" value="30"/>
  </node>
  <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" args="/usb_cam/image_raw" required="true" />
  <node name="face_search_node" pkg="myface_recognition" type="face_search_node.py" required="true" output="screen" />
</launch>
